doc/api/perf_hooks.md

```diff
@@ -112,6 +112,16 @@ added: REPLACEME
 Starts monitoring the event loop for delays. The accumulated event loop delay
 is reported as `performance.nodeTiming.eventLoopDelay`.
 
+The accumulated event loop delay is determined by calculating how long the
+previous turn of the event loop lasted and adding that period to the total if
+it exceeds a given threshold determined by the sample `resolution`.
```

Comment:
This sentence is long and difficult to understand. I'd offer a suggestion for improvement, but I'm having a hard time understanding this. Is the previous event loop turn's duration only added to total if the previous event loop turn's duration exceeds `resolution`? Or is it trying to say something else?

Reply:
Oh, I see, more text below to help clarify....

Reply:
Maybe this?:

```markdown
If the duration of a turn of the event loop exceeds a threshold, it is added to
the total thus far. The threshold varies based on `resolution` according to the
following formula: 
```